test/include.js

```diff
@@ -32,23 +32,20 @@ test('loads a thrift file that imports synchronously', function t(assert) {
     });
     var importedThrift = mainThrift.modules.common;
 
-    var typeImportedByMainThrift = mainThrift
-        .types
-        .BatchGetResponse
+    var typeImportedByMainThrift = mainThrift // Thrift
+        .models
+        .BatchGetResponse // ThriftStruct
         .fieldsByName
-        .items
-        .valueType
-        .rw
```

Comment:
A bugfix in List (included here) makes it unnecessary to go through .rw to get the .valueType. I have annotated what you have at each stage of the chain to the right.